NOTICE

```diff
@@ -22,7 +19,7 @@ mccabe 0.6.1 : MIT License
 mypy 0.961 : MIT License
 mypy-extensions 0.4.3 : MIT License
 Packaging 21.3 : (BSD 2-clause "Simplified" License OR Apache License 2.0)
-pathspec 0.9.0 (https://github.com/cpburnz/python-pathspec) : Mozilla Public License 2.0
+pathspec 0.9.0 (https://github.com/cpburnz/python-pathspec) : Mozilla Public License 2.0
```

Comment:
(What changed on this line?)

Reply:
The generated NOTICE file adds a character displaying as '^M' at the end of each line. Maybe it's a new line. I manually copied this line here since we made manual changes to the original file here because the MPL licence requires some kind of attribution. I will probably copy it in the future as well so I think it's good to solve it once and for all with this MR. The process will always be the same.

Reply:
Thanks, ^M sounds to me like a \r or something, i.e. windows/dos line ending.